src/shared/common/AccessibilityInfoModal.js

```diff
@@ -0,0 +1,168 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import Modal from 'react-native-modalbox';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { Text, View, StyleSheet, Image } from 'react-native';
+import { Trans } from 'react-i18next';
+
+import { COLOR_DEEP_BLUE } from '../constants';
+import Button from './Button';
+
+/* eslint-disable global-require */
+const GLOBAL = require('../Globals');
+
+const styles = StyleSheet.create({
+    modal: {
+        height: GLOBAL.SCREEN_HEIGHT < 500 ? GLOBAL.SCREEN_HEIGHT - 50 : 'auto',
```

Comment:
What is the logic behind this? What happens if height is zero or negative?

Reply:
I don't think we need set modal  height according to the device, we can simply use `height: 'auto'` in this case.

@tnagorra what do you think?

Reply:
I don't have much idea about the usecase here.